packages/core/src/utils/components.ts

```diff
@@ -10,7 +10,7 @@ const structureComponents = new Set([
 export const isContentfulStructureComponent = (componentId?: string) =>
   structureComponents.has(componentId ?? '');
 
-export const isComponentAllowedOnRoot = (componentId: string) =>
+export const isComponentAllowedOnRoot = (componentId?: string) =>
```

Comment:
Mainly for convenience.

I added a new call to this method as `isComponentAllowedOnRoot(draggedBlockId)` where the `draggedBlockId` type is `string | undefined` and it was cleaner than calling it like `draggedBlockId && isComponentAllowedOnRoot(draggedBlockId)`

Since this function checks for exact matches to strings, an `undefined` value passed to the `componentId` won't be an issue since it will never match the strings of structure components or the divider component's id